src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -2099,6 +2099,18 @@
             }
             return "/uhgroupings";
         };
+
+        $scope.clickCheckBox = function (i) {
+            $scope.membersInCheckboxList[i.uhUuid] = !$scope.membersInCheckboxList[i.uhUuid];
+        };
```

Comment:
Can you move the `if (event.keyCode === 13 )` condition into this function just like you did with `$scope.clickWithEnter()`? Then in places in HTML that have: `ng-keydown="$event.keyCode === 13 ? clickCheckBox(i): null"`, you can get rid of `$event.keyCode === 13`.

Reply:
I made the changes.